practice-one/src/Components/Section/Navbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from '../Common/Buttons';

```

Comment:
it should be ../Common/Button not ../Common/Buttons

Reply:
Thanks! Change file Button name and update link name